### DISCUSSION TITLE
繁體中文支援

### DISCUSSION THREAD

**Discussion by aunight** [Moonvy/figmaEX#16] [General]
如題 希望可以支援繁體中文

**Comment by yArna** [comment]
希望可以有繁體中文经验者来翻译，你可以帮我们吗，https://github.com/Moonvy/figmaEX/blob/master/language/zh-hk/lang.json

**Reply by aunight** [reply]
好的

**Reply by aunight** [reply]
[lang.zip](https://github.com/Moonvy/figmaEX/files/6855654/lang.zip)

**Reply by yArna** [reply]
非常感谢你的翻译！
你是 http://www.aunight.pw/# 吗？

**Reply by aunight** [reply]
是的這是我的個人網站

**Reply by yArna** [reply]
https://github.com/Moonvy/figmaEX/releases/tag/v1.2.3 最新版本已经加入了
